src/github.rs

```diff
@@ -292,6 +292,18 @@ pub struct Comment {
     pub user: User,
     #[serde(alias = "submitted_at")] // for pull request reviews
     pub updated_at: chrono::DateTime<Utc>,
+    #[serde(rename = "state")]
+    pub pr_review_state: PullRequestReviewState,
+}
+
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+#[derive(Debug, serde::Deserialize)]
+pub enum PullRequestReviewState {
+    Approved,
+    ChangesRequested,
+    Commented,
+    Dismissed,
+    Pending,
```

Comment:
I *think* this is correct, but I'd appreciate if this was double checked before merging